site/labs/gates/basic-gates.rst

```diff
@@ -82,6 +82,8 @@ For these questions, first create the truth table if one is not already provided
 
 #. Using only ``OR`` and ``NOT``, create a circuit that always outputs ``1``, regardless of the input
 
+    * Do not use a voltage source or a ground component
```

Comment:
Seems unnecessary. Voltage and ground are both not in the set of `{OR, NOT}`. If students want to not follow instructions, they're free to not receive marks.

Reply:
Although I agree, I don't say they can use wires, inputs, and output components. This makes it more explicit.

Reply:
In other words, they're still getting used to what's going on, so this helps.

Reply:
Not sure I buy that last point. The instruction already says use OR and NOT. Even without clarity on what those are yet, it's pretty obvious that something else with a different name is not one of those.
Arguing that wire then can't be used is being full on pedantic. The text is already clear as to what it is asking for. 

Selfishly, I wouldn't want to work will an intern that requires this level of hand holding (2nd years are the youngest interns I've seen). There's good filters for that though -- grades.